astropy/cosmology/tests/test_flrw.py

```diff
@@ -79,7 +80,8 @@ def test_H0(self, cosmo_cls, cosmo):
         # validation
         assert cosmo_cls.H0.validate(cosmo, 1) == 1 * unit
         assert cosmo_cls.H0.validate(cosmo, 10 * unit) == 10 * unit
-        with pytest.raises(ValueError, match="H0 is a non-scalar quantity"):
+        with pytest.raises(ValueError,
+                           match=re.escape("H0=[1. 2.] km / (Mpc s) is a non-scalar quantity")):
```

Comment:
A bit overkill? Now you are testing quantity output too. I'd just match on "is a non-scalar"...

Reply:
SGTM!